shared/__mocks__/feature-flags.tsx

```diff
@@ -23,7 +23,7 @@ const ff: FeatureFlags = {
   stellarExternalPartners: false,
   tabletSupport: false,
   teamInvites: false,
-  teamsRedesign: false,
+  teamsRedesign: true,
```

Comment:
probably don't want this turned on in master yet

Reply:
Even for stories? I think it is more useful for stories to capture the new work and changes it has on other new work than to make sure it doesn't impact the old work too much.

Reply:
oh whoops misread the file name here, you're right!